tasks/main.yml

```diff
@@ -55,6 +57,42 @@
     group: "{{ username }}"
     mode: 0600
 
+#
+# Install an SSH keypair for the VNC user.  This will enable Guacamole
+# to perform file transfers to/from this instance.
+#
+# Note that Guacamole currently only supports RSA keys.
```

Comment:
If we want to be more specific we could say "Note that Guacamole (guacd, as of v.1.1.0) only supports RSA keys."

Reply:
See commit d954293.